css/style.css

```diff
@@ -104,3 +121,530 @@ body {
+  display: flex;
+  flex-direction: column;
+}
+
+.first_card {
+  width: 327px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  border-radius: 16px;
+  margin: 116px auto 0;
+}
+
+.first_card img {
+  padding-top: 16px;
+  width: 295px;
+  height: 220px;
+  margin-left: 16px;
+}
+
+.first_card h2 {
+  width: 295px;
+  height: 36px;
+  margin-left: 16px;
+  font-family: 'poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 8px;
+}
+
+.first_card h3 {
+  width: 55px;
+  height: 16px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: 'poppins', sans-serif;
+  color: #344563;
+  margin-left: 16px;
+  margin-top: -8px;
+}
+
+.first_card .birth {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  display: flex;
+  justify-content: space-evenly;
+  margin-left: -12px;
+  margin-top: -27px;
+}
+
+.first_card .year {
+  margin-left: -45px;
+}
+
+.first_card p {
+  width: 295px;
+  height: 76px;
+  font-family: 'poppins1', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 16px;
+}
+
+.first_card button {
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px  solid #396df2;
+  padding: 8px;
+  border-radius: 10px;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  margin-left: 16px;
+  margin-top: 30px;
+}
+
+.second_card {
+  width: 327px;
+  height: 596px;
+  background-color: #fff;
+  margin: 116px auto 0 auto;
+  border-radius: 16px;
+}
+
+.second_card img {
+  padding-top: 16px;
+  width: 295px;
+  height: 220px;
+  margin-left: 16px;
+}
+
+.second_card h2 {
+  width: 295px;
+  height: 36px;
+  margin-left: 16px;
+  font-family: 'poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 8px;
+}
+
+.second_card h3 {
+  width: 55px;
+  height: 16px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: 'poppins', sans-serif;
+  color: #344563;
+  margin-left: 16px;
+  margin-top: 70px;
+}
+
+.second_card .birth {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  display: flex;
+  justify-content: space-evenly;
+  margin-left: -12px;
+  margin-top: -28px;
+}
+
+.second_card .year {
+  margin-left: -45px;
+}
+
+.second_card p {
+  width: 295px;
+  height: 76px;
+  font-family: 'poppins1', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 20px;
+  margin-left: 16px;
+}
+
+.second_card .button li {
+  font-size: 10px;
+  font-weight: 500;
+  line-height: 16px;
+  list-style-type: none;
+  padding: 4px 12px;
+  margin: auto;
+  border-radius: 10px;
+  background-color: #ebebff;
+  color: #6070ff;
+  display: inline-block;
+}
+
+.second_card button {
```

Comment:
[OPTIONAL]
Hi, thank you for your submission. I have a suggestion on your code:

Your code is not very DRY (Don’t Repeat Yourself). You have repeated the same properties and values for different classes of buttons, such as the font-size, font-weight, line-height, color, border, padding, border-radius, display, text-align, align-items, letter-spacing, and background-color. This makes your code longer and harder to maintain. You could use variables or inheritance to avoid repeating yourself and make your code more concise and consistent. For example, you could create a variable for the common color and use it for all the buttons that need it. Or you could create a parent class for the common properties and values and use it for all the buttons that share them.
Please consider this an optional suggestion. If you have any questions or need any help, feel free to reach out to me. Thank you for your hard work and cooperation.